README.md

```diff
@@ -38,6 +38,16 @@ you need to set the OPENAI_API_TYPE, OPENAI_API_BASE and OPENAI_API_VERSION.
 The OPENAI_API_TYPE must be set to 'azure' and the others correspond to the properties of your endpoint.
 
 
+### Setup self-hosted models (vLLM)
+
+Export huggingface id or path to the tokenizer and openai env vars. Unset `OPENAI_API_TYPE` and `OPENAI_API_VERSION`.
+
+    export TOKENIZER_PATH=<PATH_TO_TOKENIZER>
```

Comment:
What is an expected value for TOKENIZER_PATH ?

Reply:
Your `huggingface id or path to the model weights/tokenizer`. I will add an example to clarify that.